src/index.js

```diff
@@ -0,0 +1,7 @@
+import './index.scss';
+import { app } from './app';
+
+const element = app();
+element.forEach(element => {
+  document.body.appendChild(element);
+});
```

Comment:
```suggestion
const elements = app();
elements.forEach(element => {
  document.body.appendChild(element);
});
```
just a suggestion, it clarifies that `app()` returns an array of elements